### DISCUSSION TITLE
JPMC Signing-in

### DISCUSSION THREAD

**Discussion by roberthuarcaya** [jpmorganchase/salt-ds#2670] [Q&A] [answered]
I am getting the following error, 
"Your organization requires SAML single sign-on." 

How can I get access to the list of components?
https://www.figma.com/login?is_not_gen_0=true&resource_type=folder

**Comment by joshwooding** [comment, marked as answer]
Hi @roberthuarcaya, Do you mind contacting salt.design.system@jpmorgan.com with your internal email please.
